### PR TITLE
Reenable stdlib tests from #3095 

### PR DESCRIPTION
**Pull request by nstott**
reenable stdlib tests that were removed, as per #3095 

I think this is all the tests from #3095 reenabled, that issue could probably be closed

### CODE REVIEW

**Comment by nstott** [comment]
gross, it failed because our install scripts are expecting a `deno` to exist in our path, and it doesn't in this environment.

`/Users/runner/.deno/bin/echo_test: line 13: deno: command not found`

I worked around it by just setting our `PATH=../target/debug/` to find the deno executable we just built.  kind of gross, any other ideas?

File: std/installer/test.ts [hunk 1]
```diff
@@ -12,6 +12,9 @@ import { install, isRemoteUrl } from "./mod.ts";
 let fileServer: Deno.Process;
 const isWindows = Deno.build.os === "win";
```
Comment (nayeemrmn) [inline std/installer/test.ts]: I didn't get a chance to change this because the code wasn't there, but could you use `path.isWindows` instead?

Comment (nstott) [reply std/installer/test.ts]: if i get it working :)

**Comment by nstott** [comment]
in order for the install tests to run, we need a deno binary that's available to the tests, 

we have one that exists in `target/release`, but it's not available to the installer scripts.  it seems reasonable for the build environment to ensure that we can run that binary.  

making the tests responsible for the manipulating the environment to add that deno binary seems sub-optimal.

File: std/prettier/main_test.ts [hunk 2]
```diff
@@ -226,20 +225,35 @@ test(async function testPrettierPrintToStdout(): Promise<void> {
 
   const { stdout } = await run([...cmd, file0]);
   // The source file will not change without `--write` flags.
-  assertEquals(await getSourceCode(file0), "console.log (0)" + EOL);
+  assertEquals(
+    await getSourceCode(file0),
+    `console.log (0)
+`
```
Comment (nstott) [inline std/prettier/main_test.ts]: this backtick string pattern is used here https://github.com/nstott/deno/blob/reenable-stdlib-tests/std/prettier/main_test.ts#L190-L191 and seems to work on all arches
testing for EOL on windows builds is odd, for example in this test

`assertEquals(await getSourceCode(file0), "console.log (0)" + EOL);`

`"console.log (0)" + EOL` has a different value on windows builds than on linux/mac, but the contents of the file don't change.

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -63,16 +63,19 @@ jobs:
         if: startsWith(matrix.os, 'ubuntu')
         run: |
           echo ::add-path::`pwd`/prebuilt/linux64
+          echo ::add-path::`pwd`/target/release
```
[the remark this answers is not in the document]
Comment (ry) [reply .github/workflows/build.yml]: Thanks. I've added a TODO to the code.

**Pull request merged**